runtime/client_http_request_test.go

```diff
@@ -192,7 +192,7 @@ func TestBarClientWithoutHeaders(t *testing.T) {
 
 	logs := gateway.AllLogs()
 
-	assert.Equal(t, 1, len(logs))
+	assert.Equal(t, 3, len(logs))
```

Comment:
These `assert.Equal(t, 1, len(logs))` tests aren't very good tests because they check for library log output. If any dependent libraries emit extra log lines (which they are allowed to do at any time), these tests would break. Should I just remove them?

Reply:
I agree with you both. We should remove the number checking tests.